src/libertem/analysis/base.py

```diff
@@ -249,6 +266,29 @@ def get_parameters(self, parameters):
         """
         raise NotImplementedError()
 
+    @classmethod
+    def get_analysis_by_type(cls, type_):
```

Comment:
Why is this needed? Shouldn't the registry be used for this?

Reply:
Oops! Download feature was a different branch merged into this. I must have done something wrong while merging.

Reply:
Ah, yeah, that happens :)